docs/index.html

```diff
@@ -0,0 +1,5 @@
+<html>
+  <head>
+	  <meta http-equiv="refresh" content="0; url=/v0.2.0">
```

Comment:
is this generated or will someone need to update this as well as the `VERSION` file value during every release ?

Reply:
The VERSION is generated too, so I think maybe we can generate this file too. Or we can just point to the latest not the current release

Reply:
Can we just take [this approach](https://github.com/envoyproxy/gateway/pull/639/files#diff-b5b4b7207c61dcd99aa1033537dd2f0a81bb0aef3dd79d663718c67ba396c1a8R9) to update this value?